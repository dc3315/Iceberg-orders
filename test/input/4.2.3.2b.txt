S,100323,101,20000
B,100324,99,50000
B,100325,98,25500
B,100326,100,72500,10000
S,100327,100,11000
B,100328,100,50000,20000
S,100329,100,35000