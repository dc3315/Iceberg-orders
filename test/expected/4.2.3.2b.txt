+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|          |             |       |    101|       20,000|    100323|
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100324|       50,000|     99|    101|       20,000|    100323|
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100324|       50,000|     99|    101|       20,000|    100323|
|    100325|       25,500|     98|       |             |          |
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100326|       10,000|    100|    101|       20,000|    100323|
|    100325|       25,500|     98|       |             |          |
|    100324|       50,000|     99|       |             |          |
+----------+-------------+-------+-------+-------------+----------+

100326,100327,100,11000
+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100326|        9,000|    100|    101|       20,000|    100323|
|    100325|       25,500|     98|       |             |          |
|    100324|       50,000|     99|       |             |          |
+----------+-------------+-------+-------+-------------+----------+

+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100326|        9,000|    100|    101|       20,000|    100323|
|    100328|       20,000|    100|       |             |          |
|    100324|       50,000|     99|       |             |          |
|    100325|       25,500|     98|       |             |          |
+----------+-------------+-------+-------+-------------+----------+

100326,100329,100,15000
100328,100329,100,20000
+-----------------------------------------------------------------+
| BUY                            | SELL                           |
| Id       | Volume      | Price | Price | Volume      | Id       |
+----------+-------------+-------+-------+-------------+----------+
|    100326|        4,000|    100|    101|       20,000|    100323|
|    100328|       20,000|    100|       |             |          |
|    100324|       50,000|     99|       |             |          |
|    100325|       25,500|     98|       |             |          |
+----------+-------------+-------+-------+-------------+----------+

